### PR TITLE
Add prefer_schema1_digest param to koji_upload

### PR DESCRIPTION
**Pull request by lcarva**
Signed-off-by: Luiz Carvalho <lucarval@redhat.com>

### CODE REVIEW

File: atomic_reactor/plugins/post_koji_upload.py [hunk 1]
```diff
@@ -96,6 +96,8 @@ def __init__(self, tasker, workflow, kojihub, url, build_json_dir,
         :param koji_principal: str, Kerberos principal (must specify keytab)
         :param koji_keytab: str, keytab name (must specify principal)
         :param blocksize: int, blocksize to use for uploading files
+        :param prefer_schema1_digest: bool, when True, schema v1 digest is
+            will be preferred as the built image digest
```
Comment (bfontecc007) [inline atomic_reactor/plugins/post_koji_upload.py]: Little grammar nitpick: "is will be preferred"

Comment (lcarva) [reply atomic_reactor/plugins/post_koji_upload.py]: Thanks! fixed.

File: atomic_reactor/plugins/post_koji_upload.py [hunk 2]
```diff
@@ -96,6 +96,8 @@ def __init__(self, tasker, workflow, kojihub, url, build_json_dir,
         :param koji_principal: str, Kerberos principal (must specify keytab)
         :param koji_keytab: str, keytab name (must specify principal)
         :param blocksize: int, blocksize to use for uploading files
+        :param prefer_schema1_digest: bool, when True, schema v1 digest will
```
Comment (twaugh) [inline atomic_reactor/plugins/post_koji_upload.py]: worth spelling it out as 'v2 schema 1'?

Comment (lcarva) [reply atomic_reactor/plugins/post_koji_upload.py]: In the description or the parameter name?

Comment (twaugh) [reply atomic_reactor/plugins/post_koji_upload.py]: The description is enough I think.

Comment (lcarva) [reply atomic_reactor/plugins/post_koji_upload.py]: Done.

**Comment by lcarva** [comment]
Release notes updated.

**Pull request merged**